### DISCUSSION TITLE
A command output / script result as a line on the panel?

### DISCUSSION THREAD

**Discussion by srlemke** [lxqt/lxqt#2066] [Ideas]
I have a command that I regularly use, it basically switches my audio output between notebook onboard && external USB headphone, sometimes I'm not sure on which output I am, so having a panel hint, could be useful, also for other scenarios, eg. grepping some specific log(ssh, temp, other personal script, httpd...) IIRC there is something like this on KDE but I'm unsure.
Is there something like this on lxqt?

Thanks!

**Comment by tsujan** [comment]
Panel → Widgets → Add → Custom Command

**Reply by srlemke** [reply]
Oh, I think my distro may not have it enabled on build as I cannot find this one to add, thanks for the tip, will check if that's the case and maybe try a PR with it enabled, thanks!

**Reply by tsujan** [reply]
It's in git. Please wait for the next release of LXQt.
